src/components/fields/image-upload/image-review/image-thumbnails/image-thumbnails.styles.ts

```diff
@@ -101,18 +100,18 @@ export const AddImageButton = styled(IconButton)`
 	height: 3rem;
 	background: #fff;
 	${({ theme }) => {
-		const borderColor = V2_Color.Primary({ theme });
+		const borderColor = Colour["border-primary"]({ theme });
 		return `background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' rx='4' ry='4' stroke='${borderColor.replace(
 			"#",
 			"%23"
 		)}FF' stroke-width='3' stroke-dasharray='5%2c1.55' stroke-dashoffset='5' stroke-linecap='butt'/%3e%3c/svg%3e");`;
 	}}
```

Comment:
suggest to replace this with `Border.Util["dashed-default"]` from the [Border](https://designsystem.life.gov.sg/react/index.html?path=/docs/foundations-border-introduction--docs) tokens